wolox-books/src/app/screens/auth/home/screens/book-list/components/home/home.component.ts

```diff
@@ -7,9 +7,13 @@ import { Book } from 'src/app/models/book';
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss']
 })
+
 export class HomeComponent implements OnInit {
   constructor(private booksService: BooksService) { }
 
+  searchBarPlaceHolderTxt: string = 'Search books by title...';
```

Comment:
no need to define the type if you initialize the constant, typescript will infer it